tachyon/math/polynomials/multivariate/multilinear_extension_ops.h

```diff
@@ -163,8 +163,13 @@ class MultilinearExtensionOp<MultilinearDenseEvaluations<F, MaxDegree>> {
       // 0 / g(x)
       return self;
     }
-    // TODO(chokobole): Check division by zero polynomial.
-    // f(x) / 0 skips this for loop.
+    if (r_evaluations.empty()) {
+      // f(x) / 0
+      // TODO(chokobole): It should return std::nullopt.
+      // See https://github.com/kroma-network/tachyon/issues/76.
+      return MultilinearExtension<D>::Zero();
+    }
```

Comment:
I think it's better to check `r_evaluation.empty()` before `l_evaluation.empty()`.